src/diffusers/loaders/lora_pipeline.py

```diff
@@ -2278,15 +2278,15 @@ def unfuse_lora(self, components: List[str] = ["transformer", "text_encoder"], *
         super().unfuse_lora(components=components)
 
     # We override this here account for `_transformer_norm_layers`.
-    def unload_lora_weights(self):
+    def unload_lora_weights(self, reset_to_overwritten_params=False):
```

Comment:
Not too fixated on the name. Open to suggestions. 

Also open to tackling the problem in better ways.